.github/workflows/test.yml

```
@@ -133,6 +134,6 @@ jobs:
     - name: Print out version
       run: arm-none-eabi-gcc --version
     - name: Check the version matches
-      run: arm-none-eabi-gcc --version | grep -q 12.3.Rel1
+      run: arm-none-eabi-gcc --version | grep -qi 13.2.Rel1
     - name: Check output path is correct
-      run: echo "${{ steps.action-under-test-id.outputs.path }}" | grep -q /home/runner/gcc-arm-none-eabi-12.3.1-linux/arm-gnu-toolchain-12.3.rel1-x86_64-arm-none-eabi/bin
+      run: echo "${{ steps.action-under-test-id.outputs.path }}" | grep -q /home/runner/gcc-arm-none-eabi-13.2.1-linux/arm-gnu-toolchain-13.2.rel1-x86_64-arm-none-eabi/bin
```

A bit of a pain the the releases are never that consisten, but I think this should do it:
```suggestion
      run: echo "${{ steps.action-under-test-id.outputs.path }}" | grep -q /home/runner/gcc-arm-none-eabi-13.2.1-linux/arm-gnu-toolchain-13.2.Rel1-x86_64-arm-none-eabi/bin
```

---

This is the last one and we have to squash anyway, so I'll merge this commit to the PR from here and merge the PR when we finally get the all-green.